src/shipping/strategies/amazon-pay-v2/amazon-pay-v2-shipping-initialize-options.ts

```diff
@@ -1,12 +1,17 @@
 /**
  * A set of options that are required to initialize the shipping step of
- * checkout in order to support Amazon Pay V2.
+ * checkout in order to support AmazonPayV2.
+ *
+ * When AmazonPayV2 is initialized, a change shipping button will be bound.
+ * When the customer clicks on it, they will be redirected to Amazon to
+ * select a different shipping address.
  */
 export default interface AmazonPayV2ShippingInitializeOptions {
     /**
-     * This editAddressButtonId is used to set an event listener, provide an element ID if you want
-     * users to be able to select a different shipping address by clicking on a button.
-     * It should be an HTML element.
+     * This editAddressButtonId is used to set an event listener, provide an
+     * element ID if you want users to be able to select a different shipping
+     * address by clicking on a button. It should be an HTML element.
+     * @alpha
```

Comment:
I don't think you need to specify alpha in every single attribute, since the parent is already flagged as alpha in here https://github.com/bigcommerce/checkout-sdk-js/pull/910/files#diff-f42ac23e5aaa52610da2ad0590d0be18R41.

You can keep as this now, but for next time should be enough by flagging the entry points as alpha.

Reply:
Oops, I didn't know. Documentation says _The release tag applies recursively to members of a container (e.g. class or interface)._ But nothing about members. 🤔 I'd keep it in mind next time.